### PR TITLE
wrap addTrack/removeTrack

### PR DESCRIPTION
**Pull request by fippo**
wraps addTrack and removeTrack

(since i move to the native variants in Firefox)

### CODE REVIEW

File: rtcstats.js [hunk 1]
```diff
@@ -212,6 +212,16 @@ module.exports = function(wsURL, getStatsInterval, prefixesToWrap) {
         }
       });
 
+      ['addTrack', 'removeTrack'].forEach(function(method) {
```
Comment (juandebravo) [inline rtcstats.js]: you can include those two method names in the array & iteration in line 191 (`['createDataChannel', 'close']`)

Comment (fippo) [reply rtcstats.js]: done, thanks!

**Comment by juandebravo** [comment]
LGTM 💯 , travis is complaining due to lint issues

**Comment by fippo** [comment]
there is a reason I like pre-push hooks ;-)
Good thing we dont have a line-length limit

**Comment by fippo** [comment]
ah... this needs to serialize the track like addStream does :-/

**Pull request merged**